src/wled/wled.py

```diff
@@ -372,6 +372,16 @@ async def preset(self, preset: int) -> None:
         """
         await self.request("state", method="POST", data={"ps": preset})
 
+    async def live(self, live: int) -> None:
```

Comment:
Should we accept a ENUM with the different modes?

Reply:
That does seem like a better option! I can implement that later this evening probably 👍

I'm still not sure why those tests aren't passing... do you have any idea?

Reply:
Haven't looked yet, I don't see an obvious reason on first sight.